### DISCUSSION TITLE
syndicate buyback program

### DISCUSSION THREAD

**Discussion by deltanedas** [space-wizards/space-station-14#15739] [Ideaguys]
what if a salvage/cargo tech job item that will teleport traitor items\* to the syndicate in exchange for a small amount of tc, maybe 25% when you use it on something

\* should account for used implanters being worthless, emag inventory like syndie smokes, etc

say you get the station wreck and loot the c20r stuff, syndicate pays you say 10tc for the two of them and the mags

or maybe a loneop flukes you can go sell his sus box, sus toolbox, etc

would it be cool or nah

**Comment by Devaluated** [comment]
I think it would be cool to do that idea with a modified Fulton that cargotechs can buy. 

saves coding effort and adds a level of risk to sending things back. 

also fits the job better I think.

**Comment by Androclast** [comment]
Maint raiders winning from susbox maint loot

**Reply by deltanedas** [reply]
1tc revenue cargo syndies stay on top

**Reply by Androclast** [reply]
Maint raiders still winning even harder

**Comment by Hmeister-fake** [comment]
This would be sick, especially with the objective to kill a fellow syndicate
